models/recruiters.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * This file contains wrapper functions to interact with the DB.
+ * This will contain the DB schema if we start consuming an ORM for managing the DB operations
+ */
+
+const firestore = require('../utils/firestore')
+const recruiterModel = firestore.collection('recruiters')
+const userModel = firestore.collection('users')
+
+/**
+ * Add the recruiter data
+ *
+ * @param recruiterData { Object }: Recruiter data object to be stored in DB
+ * @param username { String }: Username String to be used to fetch the about user
+ * @return {Promise<{message: string, id: string, recruiterName: string, userName: string, timestamp: string }>}
+ */
+
+const addRecruiterInfo = async (recruiterData, username) => {
+  try {
+    // Add the recruiter data in DB
+    const recruiterInfo = await recruiterModel.add(recruiterData)
+    // Fetch the recruiter from DB
+    const recruiter = await recruiterModel.doc(recruiterInfo.id).get()
+    // Fetch the user from DB
+    const user = await userModel.where('username', '==', username).limit(1).get()
```

Comment:
@Ankita2002-Fr , we can use the fetchUser function here: https://github.com/Real-Dev-Squad/website-backend/blob/develop/models/users.js#L88

Reply:
I think, the fetchUser function is changing the username as null.
So I am not able to fetch the data for null username.
Can you please assist me on this.

Please check the screenshot below.

![Capture](https://user-images.githubusercontent.com/75256312/111032261-5177c600-8431-11eb-836c-99ab22100e6a.PNG)

Reply:
No worry, I have figured it out!